python/test/unit/fem/test_numba_assembly.py

```diff
@@ -6,21 +6,18 @@
 #
 # SPDX-License-Identifier:    LGPL-3.0-or-later
 
-from dolfin import (cpp, UnitSquareMesh, MPI, FunctionSpace, dx, dot, grad,
-                    TestFunction, TrialFunction, list_timings, TimingType)
+from dolfin import (cpp, UnitSquareMesh, MPI, FunctionSpace,
+                    dx, dot, grad, TestFunction, TrialFunction, list_timings,
+                    TimingType)
 from dolfin.la import PETScMatrix, PETScVector
 from numba import cfunc, types, carray, typeof
-import numpy as np
-
 from dolfin.jit.jit import ffc_jit
-
-# Define scalar_type here (change to np.complex128 for complex)
-# e.g. if has_petsc_complex(): scalar_type = np.complex128
-scalar_type = np.float64
+import numpy as np
+from petsc4py.PETSc import ScalarType
```

Comment:
Can anybody confirm that `petsc4py` is a required dependency? @chrisrichardson @garth-wells

Reply:
I assumed that petsc4py is a required dependency due the closed issue [73](https://github.com/FEniCS/dolfinx/issues/73), and it's in the list of required dependencies in [installation.rst](https://github.com/FEniCS/dolfinx/blob/ed9064cb3151f1804e71128f3c46e5cd0f94bd49/python/doc/source/installation.rst).
If it is not, I'll change back to the old code.

Reply:
I see, I was just confused by a mess which I just reported https://github.com/FEniCS/dolfinx/issues/125